### ISSUE TITLE
--style option won't change configuration

### ISSUE THREAD

**Issue by SergeySolonko** [angular/angular-cli#903]
If you'll specify ```--style``` option CLI won't save it in ```angular-config.json``` file:

1. Mac OSX El Capitan
2. angular-cli: 1.0.0-beta.5 node: 5.7.1 os: darwin x64 3.
```
ng new web-app -sn -sb --style sass
ng g r login
```
4.
```
$ ng g r login
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
installing route
installing component
  create src/app/+login/login.component.css
  create src/app/+login/login.component.html
  create src/app/+login/login.component.spec.ts
  create src/app/+login/login.component.ts
  create src/app/+login/index.ts
  create src/app/+login/shared/index.ts
```
5.
```
{
  "project": {
    "version": "1.0.0-beta.5",
    "name": "web-app"
  },
  "apps": [
    {
      "main": "src/main.ts",
      "tsconfig": "src/tsconfig.json",
      "mobile": false
    }
  ],
  "addons": [],
  "packages": [],
  "e2e": {
    "protractor": {
      "config": "config/protractor.conf.js"
    }
  },
  "test": {
    "karma": {
      "config": "config/karma.conf.js"
    }
  },
  "defaults": {
    "prefix": "app",
    "sourceDir": "src",
    "styleExt": "css",
    "prefixInterfaces": false
  }
}
```

**Comment by monojack** [comment]
Fixed in 85c9aecfb397f8d05c8b80343530191587eba432 though it's not pushed to npm yet. You can just manually edit the `angular-cli.json` for now..

**Comment by SergeySolonko** [comment]
Good, I'll close ticket then
